### PR TITLE
Add DuckDuckGo News Search and Video Search

### LINKED ISSUE #10665: Duckduckgo Video Search
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### 1. Is this request related to a challenge you're experiencing? Tell me about your story.

We need to search videos through a Duckduckgo node for our application

### 2. Additional context or comments

Tried using '!v ', but got Run failed: _extract_vqd() keywords='!v hello' Could not extract vqd.
Note that this is a potential solution: [https://github.com/langchain-ai/langchain/issues/14233](url)

### 3. Can you help us with this feature?

- [ ] I am interested in contributing to this feature.

### PR DESCRIPTION
**Pull request by taowang1993**
# Summary

This PR resolves #10665 (duckduckgo video search)

I also added duckduckgo news search.

# Video Search

image

# News Search
image

### CODE REVIEW

File: web/app/components/base/markdown.tsx [hunk 1]
```diff
@@ -265,7 +265,7 @@ export function Markdown(props: { content: string; className?: string }) {
             }
           },
         ]}
-        disallowedElements={['script', 'iframe', 'head', 'html', 'meta', 'link', 'style', 'body']}
+        disallowedElements={['script', 'head', 'html', 'meta', 'link', 'style', 'body']}
```
Comment (crazywoola) [inline web/app/components/base/markdown.tsx]: As I said, I don't think this is a good idea to remove iframe from the markdown for now.

Comment (taowang1993) [reply web/app/components/base/markdown.tsx]: The duckduckgo-video-search feature still works without removing iframe from markdown.tsx.

If keeping iframe, then users will get a video link or a list of video links.

Should I remove the changes in markdown.tsx? 

Or can you just ignore the changes in markdown.tsx?

Comment (crazywoola) [reply web/app/components/base/markdown.tsx]: Yes, please add `iframe` back, I shall approve this pr. And please fix the lint errors.

Comment (taowang1993) [reply web/app/components/base/markdown.tsx]: I added it back. What's lint errors?

**Review by loganclark360 — APPROVED** [review]

**Pull request closed without merging**